docs/docs/100-tools/02-authoring.md

```diff
@@ -0,0 +1,108 @@
+# Authoring Tools
```

Comment:
This section doesn't mention that every arg becomes an environment variable. It can be figured out from the examples, but I think it would be better to say that explicitly.